src/native/managed/cdac/mscordaccore_universal/Legacy/IXCLRData.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.Diagnostics.DataContractReader.Legacy;
 
 internal struct CLRDataModuleExtent
 {
-    public ulong /* CLRDATA_ADDRESS */ baseAddress;
+    public ClrDataAddress baseAddress;
```

Comment:
What would an alternative where this stays as `ulong` look like? I assume that you have considered it, but rejected it.